homeassistant/components/sense/sensor.py

```diff
@@ -242,3 +248,34 @@ def __init__(
     def native_value(self) -> float:
         """Return the state of the sensor."""
         return self._device.power_w
+
+
+class SenseDeviceEnergySensor(SenseDeviceEntity, SensorEntity):
```

Comment:
I think it would be nice to use entity descriptions here. The idea of them is that you create 1 (or more) generic base class, and use entity descriptions to change the way they appear. I think it would make sense to use for the device entities. Checkout `smarty` for a recent example

Reply:
I think this would be a good change, but I'd like to implement that in a different PR along with the translation strings for other sensors